.github/workflows/autowiki.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       uses: actions/cache@v4
       with:
         path: ~/BYOND
-        key: ${{ runner.os }}-byond-${{ secrets.CACHE_PURGE_KEY }}
```

Comment:
also CACHE_PURGE_KEY was supposed to be killed off in #71529 but i guess we missed these